doc/test-framework/scorecard.md

```diff
@@ -52,6 +52,10 @@ Following are some requirements for the operator project which would be  checked
 
 **NOTE:** If your operator is non-SDK then some steps will be required in order to meet its requirements.
 
+## Exit Status
```

Comment:
Note that the doc follow a sequence of information which makes easier the understanding. So, in this way, I do not think that this section would be helpful here.  WDYT about add it after https://github.com/operator-framework/operator-sdk/blob/f0f61013dea6702466d747b9a3141f2275a5f239/doc/test-framework/scorecard.md#tests-performed? Also, o not forget to update the index.